Use PascalCase for CLU samples (#37129)
Azure/azure-sdk-for-net: sdk/cognitivelanguage/Azure.AI.Language.Conversations/samples/README.md

Conversation Analysis is a cloud-based conversational AI service that applies custom machine-learning intelligence to a user's conversational, natural language text to predict overall meaning, and pull out relevant, detailed information.

> You can work with request and response models more easily by using our [Dynamic JSON](https://aka.ms/azsdk/net/dynamiccontent) feature. The following samples demonstrate how to access properties using PascalCase idiomatic to C#.

Reviewer: I like the call-out to talk about dynamic here, but wondering if it makes sense to talk about the code style convention at the top-level of the README, before the samples are introduced?  It feels like that's a nit detail that would be discussed in the sample itself, but not outside it.

```suggestion
You can work with request and response content more easily by using our [Dynamic JSON](https://aka.ms/azsdk/net/dynamiccontent) feature.  This is illustrated in the following samples.
```

Author: Sure. I can put a similar blurb (singular, of course) into each sample.
